src/components/gltf-bundle.js

```diff
@@ -5,6 +5,7 @@ AFRAME.registerComponent("gltf-bundle", {
 
   init: async function() {
     this._addGltfEntitiesForBundleJson = this._addGltfEntitiesForBundleJson.bind(this);
+    this.baseURL = new URL(THREE.LoaderUtils.extractUrlBase(this.data.src), window.location.href);
```

Comment:
is `window.location` the right thing to use to cover `iframe`s?

Reply:
I think its just to support relative paths to gltf bundles.

Reply:
I think they are equivelent, in an iframe you would have to do `window.parent.location` to get the outer context's location (if you had permission to do so).